packages/package_info/ios/Classes/PackageInfoPlugin.m

```diff
@@ -15,8 +15,14 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
 
 - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   if ([call.method isEqualToString:@"getAll"]) {
+    NSString* displayName =
+        [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleDisplayName"];
+    // Cannot put nil into a map so return an empty string.
+    if (displayName == nil) {
+      displayName = @"";
```

Comment:
Perhaps `[NSNull null]` would be more appropriate here? It is relayed as `null` to the Dart side.

Reply:
Neat idea. I would rather not return the key, rather than returning a key with a null value though. Usually `map.containsKey('x') == (map['x'] == null)`.

I will submit this as is to fix the break and will merge a different fix later.